cookbooks/aws-parallelcluster-computefleet/test/inspec.yml

```diff
@@ -4,4 +4,4 @@ license: 'Apache-2.0'
 
 depends:
   - name: common
-    path: cookbooks/aws-parallelcluster-common/test/common
```

Comment:
Are we sure common isn't used by ImageBuilder?

Reply:
I cannot find any reference of `aws-parallelcluster-common` in the CLI code.